src/pluginops/pluginops.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2020 by Delphix. All rights reserved.
+# Copyright (c) 2021, 2023 by Delphix. All rights reserved.
```

Comment:
If any file is having a copyright from any previous year than 2021, Please don't change that it to 2021. So in this case, you can use # Copyright (c) 2020, 2023 by Delphix. All rights reserved.